apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/viderefoere-opphoer/ViderefoereOpphoerVurdering.tsx

```diff
@@ -33,16 +40,26 @@ export const ViderefoereOpphoerVurdering = ({
 }) => {
   const dispatch = useAppDispatch()
 
+  const [skalViderefoere, setSkalViderefoere] = useState<JaNei | undefined>(viderefoertOpphoer?.skalViderefoere)
   const [opphoerstidspunkt, setOpphoerstidspunkt] = useState<Date | null>(
     viderefoertOpphoer ? new Date(viderefoertOpphoer.dato) : null
   )
-  const [vilkaar, setVilkaar] = useState<VilkaarType | undefined>(viderefoertOpphoer?.vilkaar)
+  const [vilkaar, setVilkaar] = useState<string | undefined>(viderefoertOpphoer?.vilkaar)
   const [vilkaarError, setVilkaarError] = useState<string>('')
   const [begrunnelse, setBegrunnelse] = useState<string>(viderefoertOpphoer?.begrunnelse || '')
   const [setViderefoertOpphoerStatus, setViderefoertOpphoer, resetToInitial] = useApiCall(lagreViderefoertOpphoer)
   const [kravdato] = useState<string | undefined>()
 
+  const [vilkaartyper, setVilkaartyper] = useApiCall(hentVilkaartyper)
+
+  useEffect(() => {
+    setVilkaartyper(behandlingId)
+  }, [behandlingId])
```

Comment:
Bruk heller annen navngivning for api calls, trodde først at dette var vanlig react state. `[vilkaartyperResult, hentVilkaartyperRequest] = useApiCall(hentVilkaartyper)`